packages/docs/src/lang/en/components/Selects.json

```diff
@@ -67,7 +67,7 @@
     "hideSelected": "Do not display in the select menu items that are already selected",
     "itemColor": "Sets color of selected items",
     "itemDisabled": "Set property of **items**'s disabled value",
-    "items": "Can be an array of objects or array of strings. When using objects, will look for a text and value field. This can be changed using the **item-text** and **item-value** props.  Objects that have a **header** or **divider** property are considered special cases and generate a list header or divider; these items are not selectable.",
+    "items": "Can be an array of objects or array of strings. When using objects, will look for a text, value and disabled keys. This can be changed using the **item-text**, **item-value** and **item-disabled** props.  Objects that have a **header** or **divider** property are considered special cases and generate a list header or divider; these items are not selectable.",
```

Comment:
Items can have `header` or `divider` too (both `boolean`)

Reply:
I added to the example the `divider = boolean` and the `header = string` options